Wikipedia/Code/WMFArticleFooterMenuViewController.m

```diff
@@ -39,6 +39,16 @@ - (void)viewWillAppear:(BOOL)animated {
     [self.tableView deselectRowAtIndexPath:[self.tableView indexPathForSelectedRow] animated:YES];
 }
 
+- (CGFloat)tableView:(UITableView *)tableView heightForHeaderInSection:(NSInteger)section {
```

Comment:
why can't we use a plain table instead? will "about this article" ever have multiple sections?